Sources/TrustlySDK/Utils/JSONUtils.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  JSONUtils.swift
+//  TrustlySDK
+//
+//  Created by Marcos Rivereto on 08/02/24.
+//
+
+import Foundation
+
+struct JSONUtils {
+    
+    static private func getJsonDataFrom(dicticionary: [String : AnyHashable]) -> Data? {
```

Comment:
Maybe there's a typo here in the word `dictionary`?

Reply:
Fixed